bucket/questdb.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "6.0.9",
+    "description": "An open source SQL database designed to process time series data, faster.",
+    "homepage": "https://questdb.io",
+    "license": "Apache-2.0",
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/questdb/questdb/releases/download/6.0.9/questdb-6.0.9-no-jre-bin.tar.gz",
+            "hash": "b8d7b80c60be4c6e2cb969fa2245533b1db37de960f70288676543578c9181b5",
+            "extract_dir": "questdb-6.0.9-no-jre-bin"
+        }
+    },
+    "bin": "questdb.exe",
+    "persist": [
+        "conf",
+        "db",
+        "log"
+    ],
+    "checkver": {
+        "github": "https://github.com/questdb/questdb"
+    },
+    "autoupdate": {
+        "url": "https://github.com/questdb/questdb/releases/download/$version/questdb-$version-no-jre-bin.tar.gz",
```

Comment:
This should be in 64 bit block BTW

Reply:
This can update all architecture, but yes, move it under `arch` is okay.

Reply:
If it has 32 bit support then it should be outside though.

In architecture block, you had used 64 bit, so thought it was 64 bit.